src/DGtal/geometry/tools/QuickHull.h

```diff
@@ -0,0 +1,1599 @@
+/**
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ **/
+
+#pragma once
+
+/**
+ * @file QuickHull.h
+ * @author Jacques-Olivier Lachaud (\c jacques-olivier.lachaud@univ-savoie.fr )
+ * Laboratory of Mathematics (CNRS, UMR 5127), University of Savoie, France
+ *
+ * @date 2020/07/11
+ *
+ * Header file for module QuickHull.cpp
+ *
+ * This file is part of the DGtal library.
+ */
+
+#if defined(QuickHull_RECURSES)
+#error Recursive header files inclusion detected in QuickHull.h
+#else // defined(QuickHull_RECURSES)
+/** Prevents recursive inclusion of headers. */
+#define QuickHull_RECURSES
+
+#if !defined QuickHull_h
+/** Prevents repeated inclusion of headers. */
+#define QuickHull_h
+
+//////////////////////////////////////////////////////////////////////////////
+// Inclusions
+#include <iostream>
+#include <string>
+#include <vector>
+#include <queue>
+#include <set>
+#include "DGtal/base/Common.h"
+#include "DGtal/base/Clock.h"
+#include "DGtal/geometry/tools/QuickHullKernels.h"
+
+namespace DGtal
+{
+  /////////////////////////////////////////////////////////////////////////////
+  // template class QuickHull
+  /**
```

Comment:
doxygen generates "comment" blocks for this doc (because of the four " "before each line maybe). 
remove the spaces?

Reply:
done